java/clients/src/main/java/sleeper/clients/admin/deploy/UpdateExistingInstance.java

```diff
@@ -85,13 +90,18 @@ public void update() throws IOException, InterruptedException {
                 .jarsDirectory(jarsDirectory)
                 .ensureNewInstance(false).skipVersionCheck(true)
                 .build().deployInferringType(properties);
+
+        RestartTasks.builder().ecs(ecs)
+                .properties(properties)
+                .build().run();
```

Comment:
The CdkDeployInstance call could have changed the values for the ECS cluster properties.

This step is only needed if the jars have changed.